.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,9 @@
+- id: "check-for-raw-in-templates"
+  name: "Check Instances of 'raw' in a certain folder"
+  entry: check-for-raw-in-templates.sh
+  types: [text]
+  exclude: ".yaml$|.yml$"
+  language: "script"
+  description: "This hook checks for instances of raw in a folder you choose"
+  args: [--d]
+  
```

Comment:
Ironic that a pre-commit repo is missing an EOF new line checker 😂

Reply:
We're using this in digideps:

```
-   repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v2.0.0
    hooks:
    -   id: trailing-whitespace
    -   id: end-of-file-fixer
```

Reply:
Chatted off github - leaving out non-custom pre-commit hooks for now until we've investigated how this works as this needs to get merged for a Digideps ticket.